android/src/org/ppsspp/ppsspp/NativeActivity.java

```diff
@@ -296,7 +296,12 @@ public void Initialize() {
 	    File sdcard = Environment.getExternalStorageDirectory();
 
 	    String externalStorageDir = sdcard.getAbsolutePath();
-	    String dataDir = this.getFilesDir().getAbsolutePath();
+	    File filesDir = this.getFilesDir();
+		String dataDir = null;
```

Comment:
nit: Tabs and spaces living in despicable harmony.

-[Unknown]

Reply:
I know :( not gonna bother fixing though, as you say we should just rip the band-aid.